fab/environments.yml

```diff
@@ -100,7 +100,7 @@ production:
       email_queue:
         concurrency: 2
       repeat_record_queue:
-        concurrency: 50
+        concurrency: 100
```

Comment:
as we chatted about offline, if this doesn't help we'll need to just add another worker